stringtie/1.2.4/prepDE.py

```diff
@@ -52,6 +52,12 @@ def t_overlap(t1, t2): #from badGenes: chromosome, strand, cluster, start, end,
             if is_overlap(t1[i], t2[j]): return True
     return False
 
+def list_gtf_files(dir_path):
+    glob_pattern = os.path.join(dir_path, "*.gtf")
+    return [
+        x for x in glob.glob(glob_pattern)
+        if not re.search(r"[.]refs[.]gtf$", x)
+    ]
```

Comment:
This functions lists all GTF files in a directory, ignoring files whose names end with `.refs.gtf`.